tests/e2e/lib/virt_helpers.go

```diff
@@ -210,6 +222,32 @@ func (v *VirtOperator) checkHco() bool {
 	return health == "healthy"
 }
 
+// Check if KVM emulation is enabled.
```

Comment:
what is KVM?

Reply:
I think it's referring to this [qemu KVM](https://wiki.qemu.org/Features/KVM) . Read more at [wiki](https://en.wikipedia.org/wiki/Kernel-based_Virtual_Machine)

Reply:
KVM here means kernel virtual machine, which is a kernel feature that accelerates virtual machines using the virtualization extensions provided by the CPU. Most cloud clusters do not have direct access to the CPU, and can't use KVM. So this pull request turns on an emulation mode that is much slower, but will work on our CI instances.